src/sagemaker/jumpstart/converter_utils.py

```diff
@@ -0,0 +1,105 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""This module stores CuratedHub converter utilities for JumpStart."""
+from __future__ import absolute_import
+
+import re
+from typing import Any, Dict, List
+from sagemaker.jumpstart.enums import ModelSpecKwargType, NamingConventionType
+
+
+def camel_to_snake(camel_case_string: str) -> str:
+    """Converts camelCaseString or UpperCamelCaseString to snake_case_string."""
+    snake_case_string = re.sub("(.)([A-Z][a-z]+)", r"\1_\2", camel_case_string)
+    return re.sub("([a-z0-9])([A-Z])", r"\1_\2", snake_case_string).lower()
```

Comment:
don't we already have this logic somewhere? let's make sure this exists in only 1 place and is tested

Reply:
I moved it :) so that is only one.